test/setup/server_setup.go

```diff
@@ -304,6 +304,72 @@ func (s *Server) SubmitTransaction(t *testing.T, urlPath string, tx interface{})
 	return receipt, nil
 }
 
+func (s *Server) SetConfigTx(t *testing.T, newConfig *types.ClusterConfig, version *types.Version) (string, *types.TxReceipt, error) {
+	client, err := s.NewRESTClient(s.clientCheckRedirect)
+	if err != nil {
+		return "", nil, err
+	}
+
+	txID := uuid.New().String()
+	configTx := &types.ConfigTx{
+		UserId:               "admin",
+		TxId:                 txID,
+		ReadOldConfigVersion: version,
+		NewConfig:            newConfig,
+	}
+
+	// Post transaction into new database
+	response, err := client.SubmitTransaction(constants.PostConfigTx, &types.ConfigTxEnvelope{
+		Payload:   configTx,
+		Signature: testutils.SignatureFromTx(t, s.AdminSigner(), configTx),
+	}, 30*time.Second)
```

Comment:
In one of the following tests, we will want to check that only an admin user can submit a config-tx. So I think it would save some time later if you put the signer as an input argument, and pass the `s.AdminSigner()` as an argument for now. Later we will have to test it with a regular user as well

Reply:
Done